src/backend/aspen/app/views/phylo_trees.py

```diff
@@ -167,6 +167,34 @@ def sample_filter(sample: Sample):
     return json_data
 
 
+def _get_selected_samples(db_session, phylo_tree_id):
+    # SqlAlchemy requires aliasing for any queries that join to the same table (in this case, entities)
+    # multiple times via joined table inheritance
+    # ref: https://github.com/sqlalchemy/sqlalchemy/discussions/6972
+    entity_alias = aliased(UploadedPathogenGenome, flat=True)
+    phylo_tree: PhyloTree = (
+        db_session.query(PhyloTree)
+        .join(PhyloRun, PhyloTree.producing_workflow.of_type(PhyloRun))
+        .outerjoin(entity_alias, PhyloRun.inputs.of_type(entity_alias))
+        .outerjoin(Sample)
+        .filter(PhyloTree.entity_id == phylo_tree_id)
+        .options(
+            contains_eager(PhyloTree.producing_workflow.of_type(PhyloRun))
+            .contains_eager(PhyloRun.inputs.of_type(entity_alias))
+            .contains_eager(entity_alias.sample)
+        )
+        .one()
+    )
```

Comment:
Thanks for your hard work figuring out this query! I know it was not easy.

Reply:
ditto!

Reply:
ditto!